### DISCUSSION TITLE
Nouvelle version de lunii incompatible

### DISCUSSION THREAD

**Discussion by AlexTraveylan** [marian-m12l/studio#406] [General]
Bonjour,

Je viens d'acheter une lunii v3, elle est connecté en wifi et ne semble pas fonctionner avec votre logiciel.
(j'ai testé le logiciel pour les v1 et v2 ca fonctionnait bien)
Ce serait cool un patch ou si quelqu'un a une idée pour faire fonctionner STUdio avec une v3.

Bonne journée !

**Comment by azouaouu** [comment]
Meme probleme, lunii v3 non reconnu.

**Comment by antoinecoulon** [comment]
UP!! Quelqu'un a du nouveau ? Noël passé, les lunii v3 ont dû arriver chez certains !!

**Reply by mpiffault** [reply]
Pas testé pour l'instant, mais il y a un support expérimental qui est en place depuis peu sur la master, cf. https://github.com/marian-m12l/studio/issues/398#issuecomment-1872959910
